rubygem-azure-core/rubygem-azure-core.spec

```diff
@@ -0,0 +1,89 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Generated from azure-0.7.5.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name azure
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.7.5
```

Comment:
The .gem in the dir is version 0.1.1, or you could use 0.1.2.  0.7.5 is azure, not azure-core.